cypress/utils/utils.ts

```diff
@@ -798,7 +798,7 @@ export function performRowActionByIcon(itemName: string, action: string): void {
         .find(action)
         .first()
         .find("button", { log: true, timeout: 30 * SEC })
-        .first()
+        .eq(1)
```

Comment:
This change is not there in release-0.7 branch .
@midays can you please check if things are different in 7.3.1 and MTA 8 ? do we need the change at both places?

Reply:
https://github.com/konveyor/tackle-ui-tests/blob/release-0.7/cypress/utils/utils.ts#L805